pom.xml

```diff
@@ -631,8 +631,8 @@
         <version>1.1.0</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.3.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.3.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline[6.2.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```

Comment:
@ardian4 
Same as above? Is this change required in the PR, or we should do this change just while testing our plugin in CDAP 6.2 and/or CDAP 6.1 for instance?